xamarin/Framework/LiferayScreens.iOS/ImageGallery/ImageEntryUpload.cs

```diff
@@ -32,7 +32,7 @@ interface ImageEntryUpload : INSCoding
         IntPtr Constructor(UIImage image, [NullAllowed] UIImage thumbnail, string title, string notes);
 
         // -(void)encodeWithCoder:(NSCoder * _Nonnull)aCoder;
-        [Export("encodeWithCoder:")]
+        [Export("encodeWithCoderImageEntryUpload:")]
```

Comment:
This method doesn't exist, isn't it? This will crash if we call this method from xamarin?